odbc/src/storages/odbc/detail/connection.cpp

```diff
@@ -0,0 +1,103 @@
+#include <sql.h>
+#include <sqlext.h>
+#include <stdexcept>
+#include <vector>
+
+#include <storages/odbc/detail/connection.hpp>
+
+USERVER_NAMESPACE_BEGIN
+
+namespace storages::odbc {
+
+Connection::Connection(const std::string& dsn) : env_(SQL_NULL_HENV), handle_(SQL_NULL_HDBC) {
```

Comment:
Make a RAII wrapper for handle. In that way all the manual calls to `SQLFreeHandle` will go away